packages/core/src/scd-queries/ied/types.ied.d.ts

```diff
@@ -0,0 +1,56 @@
+// CONSTANTS
+import type { SCD_ELEMENTS } from '@/constants/element.constant'
+// TYPES
+import type {
+	SCDBaseElement,
+	AllowedElements,
+	SubstationElement,
+	VoltageLevelElement,
+	BayElement,
+	IEDElement,
+	LDeviceElement,
+	LNodeElement,
+	ExtRefElement
+} from '../types.scd-queries'
+
+export namespace IED {
+	export type CommunicationInfo = {
+		iedName: string
+		iedDetails: Details
+		published: PublishedMessage[]
+		received: ReceivedMessage[]
+	}
+
+	export type Details = {
+		logicalNodes: string[]
+		dataObjects: string[]
+		dataAttributes: string[]
+	}
+
+	export type PublishedMessage = {
+		id: string
+		name: string
+		targetIEDName: string
+		serviceType: string
+		serviceDatSet: string
+		serviceCbName: string
+	}
+
+	export type ReceivedMessage = {
+		iedName: string // to show
```

Comment:
to show = is displayed on the page?
to filter = you use it to search?

Reply:
Like the other comments, these comes from the legacy codebase. I'll remove them as they are confusing.